obal/__init__.py

```diff
@@ -24,17 +25,35 @@
 display = None  # pylint: disable=C0103
 
 
+class Playbook(object):
+    """
+    An abstraction over an Ansible playbook
+    """
+    def __init__(self, path):
+        self.path = path
+
+        filename = os.path.basename(path)
+        self.name = os.path.splitext(filename)[0]
+
+    @property
+    def takes_package_parameter(self):
+        """
+        Whether this playbook takes a package argument.
+
+        This is determined by a hosts: packages inside the playbook
+        """
+        with open(self.path) as playbook:
+            data = yaml.load(playbook.read())
+
+        return any('packages' in row['hosts'].split() for row in data)
```

Comment:
shouldn't `hosts` be a list anyways?

Reply:
In a playbook it's not; from our playbooks I've found the following:
```yaml
- hosts: localhost
- hosts: packages
- hosts: repoclosure packages
```

Reply:
We can (should) update to a list format in our playbooks and simplify this code and use a more yaml native construct.

Reply:
I'd be happy to use that. Can we enforce that via ansible-lint? Otherwise the automated tests will cover it as well but might be less readable. Do we want this as part of this PR?